### DISCUSSION TITLE
🔗 Add custom shapes to Elementor (Duplicate)

### DISCUSSION THREAD

**Discussion by netmedia84** [elementor/elementor#28489] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Same request: I need to use custom separator shapes, and Elementor Pro only allows the use of a few pre-installed separator shapes.

### Describe the solution you'd like.

It should be possible to add custom separator shapes in SVG format quite easily, as an additional feature to be added in the Elementor menu, similar to fonts, icons, and custom codes.
I tested a plugin that allows the addition of custom separator shapes in SVG.
https://github.com/MarieComet/elementor-custom-shapes
However, this plugin hasn’t been updated for 2 years, and you still need to manually edit the SVG code for compatibility.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
https://developers.elementor.com/docs/hooks/shapes/
